datahub/core/templates/core/docs/swagger-ui.html

```diff
@@ -27,7 +27,11 @@
           SwaggerUIBundle.presets.apis,
           SwaggerUIBundle.SwaggerUIStandalonePreset
         ],
-        layout: "BaseLayout"
+        layout: "BaseLayout",
+        requestInterceptor: (req) => {
+            req.headers['X-CSRFToken'] = "{{csrf_token}}"
```

Comment:
Is there a semicolon missing?

```suggestion
            req.headers['X-CSRFToken'] = "{{csrf_token}};"
```

Reply:
Yes! Updated. Thank you.